torch/_higher_order_ops/cond.py

```diff
@@ -298,12 +297,14 @@ def _same_meta_except_requires_grad(true_out, false_out):
                 and true_meta.stride == false_meta.stride
             )
 
+        """
         if not _same_meta_except_requires_grad(true_out, false_out):
             raise torch._dynamo.exc.CondOpArgsMismatchError(
                 f"Expected each tensor to have same metadata but got:"
                 f"\n  {true_fn.__name__} returns {true_out.meta['tensor_meta']}"
                 f"\n  {false_fn.__name__} returns {false_out.meta['tensor_meta']}"
             )
+        """
```

Comment:
This and the implementation above is OK to remove as long as merge_tensors covers all the checks we were doing previously